Directory Structure Change

Sebitosh opened the pull request:
Proposal to change the directory structure in a more modular and parameterized way.

The directories would be used as follows:

`/config_tests/` would be where the input yaml files for `generate_rules.py` would be located.

`/config_infra/` would be where we develop proposed infrastructures for setting up easily the testing environment. It is empty for now (hence why the `.gitignore`).

`/generated/` would be where we place the output of `generate_rules.py`. We would place the existing `/rules/` and `/tests/regression/test/` inside.

`/mrts/` would be the directory where the existing and future scripts would be located.

airween commented:
The only one thing (probably the most important): could you update [README.md](https://github.com/Sebitosh/MRTS/blob/sebitosh-dir-struct/README.md), especially the [Run the tool](https://github.com/Sebitosh/MRTS/blob/sebitosh-dir-struct/README.md#run-the-tool) section? It would be enough to run a test generate, copy the relevant commands and their results, then run the engine with generated rules and finally run the tests.

Sebitosh commented:
> The only one thing (probably the most important): could you update [README.md](https://github.com/Sebitosh/MRTS/blob/sebitosh-dir-struct/README.md), especially the [Run the tool](https://github.com/Sebitosh/MRTS/blob/sebitosh-dir-struct/README.md#run-the-tool) section? It would be enough to run a test generate, copy the relevant commands and their results, then run the engine with generated rules and finally run the tests.

I updated the file for the run and covered variables sections. I also added a small explanation on how to include the rules using `mrts.load` (though rudimentary for now, an improved way of doing this will come with our future propositions)
